analyzer/CMakeLists.txt

```diff
@@ -3,3 +3,15 @@
 add_subdirectory(file)
 add_subdirectory(protocol)
 
+if ( SPICY_SCRIPTS_OUTPUT_DIR_INSTALL )
+    if ( SPICY_IN_TREE_BUILD )
+        # Do not install by default but tie to install-spicy-analyzers target.
+        install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/ DESTINATION "${SPICY_SCRIPTS_OUTPUT_DIR_INSTALL}/spicy-analyzers/"
```

Comment:
`SPICY_SCRIPTS_OUTPUT_DIR_INSTALL` is constructed from `CMAKE_INSTALL_FULL_LIBDIR` which is an absolute path so this might cause issues down the line (e.g., in `DESTDIR` builds).

Can we use a relative path here? Maybe it would be possible to initialize `SPICY_SCRIPTS_OUTPUT_DIR_INSTALL` from `CMAKE_INSTALL_LIBDIR` instead?

Reply:
`SPICY_SCRIPTS_OUTPUT_DIR_INSTALL` is actually usually coming out of `spicyz --print-scripts-path` (i.e., from external). The `CMAKE_INSTALL_FULL_LIBDIR` version is actually just a fallback, not even sure we still need that (but didn't want to remove as part of this in case it would break something else).